test/test_benchmark.py

```diff
@@ -194,7 +194,7 @@ def build(self, parent, filename, args, shared_args, emcc_args, native_args, nat
       # systems (like zlib) if they see a CFLAGS it will override all their
       # default flags, including optimizations.
       env_init['CFLAGS'] = ' '.join(LLVM_FEATURE_FLAGS + [OPTIMIZATIONS] + self.extra_args)
-      emcc_args += lib_builder('js_' + llvm_root, native=False, env_init=env_init)
+      emcc_args = emcc_args + lib_builder('js_' + llvm_root, native=False, env_init=env_init)
```

Comment:
Can you add a comment here to remind folks (me) not to reduce this to `+=`?

Reply:
Done